depends/packages/qt.mk

```diff
@@ -254,6 +256,7 @@ define $(package)_preprocess_cmds
   patch -p1 -i $($(package)_patch_dir)/fast_fixed_dtoa_no_optimize.patch && \
```

Comment:
In: a995902d604c701be4f46087057b907de9a0ecca

This also appears to pull in fast_fixed_dta_no_optimise.patch and I think that may not be intentional based on the commit it's rebasing?:

![image](https://github.com/bitcoin/bitcoin/assets/6606587/172d24d9-d07a-4c13-946e-6980c8a1beb7)

Reply:
Sorry for the noise, this is just my script misbehaving.
Script confused as fast_fixed_dtoa_no_dtoa_no_optimse was removed from master in the interim which confused the diffs.